raw/era5_sl_hourly.cfg

```diff
@@ -0,0 +1,88 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[parameters]
+client=cds
+dataset=reanalysis-era5-single-levels
+# https://confluence.ecmwf.int/display/CKB/Climate+Data+Store+%28CDS%29+API+Keywords
+
+target_path=gs://ecmwf-downloads/deepmind/date-variable-single_level/{date:%%Y/%%m/%%d}/{variable}/surface.nc
```

Comment:
Please update this path, too.

Reply:
The required modifications have been made.